app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PagesController < ApplicationController
+  def admin
+    @admin_members = SlackUser.admins
```

Comment:
Let's tack the `available` scope onto this as well as that will remove admin bots and admins that have been deleted.